.gitignore

```
@@ -72,6 +72,8 @@ pip-delete-this-directory.txt
 coverage.xml
 htmlcov/
 nosetests.xml
+docrepr/tests/screenshots/**
+docrepr/tests/diffs/**
```

```suggestion
docrepr/tests/screenshots/
docrepr/tests/diffs/
```

Shouldn't be needed, as ending with a `/` should ignore the whole dir, like the others listed